colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<ul><li>Is it possible to use an ORG_NAME not associated with the bug bash? Actually wondering if we need to just use &lt;INSERT YOUR ORG_NAME HERE&gt; syntax as this will only actually work if you have a key associated with that org (please correct me if I am wrong about this).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>